draft-ietf-quic-recovery.md

```diff
@@ -371,16 +371,23 @@ retain sufficient history is an open research question.
 
 ## Estimating min_rtt {#min-rtt}
 
-min_rtt is the minimum RTT observed over the lifetime of the connection.
-min_rtt is set to the latest_rtt on the first sample in a connection, and to the
-lesser of min_rtt and latest_rtt on subsequent samples.
+min_rtt is the minimum RTT observed for a given congestion control context.
+min_rtt is set to the latest_rtt on the first RTT sample, and to the lesser
+of min_rtt and latest_rtt on subsequent samples.  min_rtt is used to reject
+implausible RTT samples, but is not used directly in congestion control or
```

Comment:
I would drop the rest of the text after "samples" and perhaps add "which would otherwise cause the smoothed_rtt estimate to be incorrectly low.". I understand what you're trying to say, but I think saying that it is not used directly in loss recovery makes it a bit confusing I think. It is used, and I think people will have different reads on what's indirect and what's